Sriram/Reg.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Registration page</title>
+  </head>
+  <body>
+    <h2><strong>Register</strong></h2>
+
+    <form id="Form">
+      First name:
+      <input type="text" name="firstname" value="First name" />
+      Last name:
+      <input type="text" name="lastname" value="Last name" /><br /><br />
+      Gender:
+      <label for="male">Male</label>
+      <input type="radio" name="gender" value="male" checked /> Male
+      <label for="female">Female</label>
+      <input type="radio" name="gender" value="female" /> Female
+      <label for="other">Other</label>
+      <input type="radio" name="gender" value="other" /> Other<br /><br />
```

Comment:
**_required_** attribute missing. And male shouldn't be checked by default as it can end up in accidental submission and also offensive to other genders in certain cases.